perfmetrics/scripts/hns_rename_folders_metrics/config.json

```diff
@@ -0,0 +1,93 @@
+{
+  "name": "hns-rename-folders-test" ,
+  "folders" : {
+    "num_folders": 3,
+    "folder_structure" : [
+      {
+        "name": "1k_files_rename_test_0" ,
+        "num_files": 1000 ,
+        "file_name_prefix": "file" ,
+        "file_size": "1kb"
+      },
+      {
+        "name": "5k_files_rename_test_0" ,
+        "num_files": 5000 ,
+        "file_name_prefix": "file" ,
+        "file_size": "1kb"
+      },
+      {
+        "name": "10k_files_rename_test_0" ,
+        "num_files": 10000 ,
+        "file_name_prefix": "file" ,
+        "file_size": "1kb"
+      }
+    ]
+  },
+  "nested_folders": {
+    "folder_name": "nested_folder_rename_test",
+    "num_folders": 10,
+    "folder_structure" :  [
+      {
+        "name": "nested_folder_1_0" ,
```

Comment:
Can you change the name of directory with `1k_files_rename_test_0` and for others

Reply:
This configuration is for [this](https://docs.google.com/document/d/1Ws_0aW_NigxsF5zkuqDb0LvHFl5itwE1JlwHmPXimEw/edit?tab=t.0#bookmark=id.kf3nigt9vp3d) test